explorer/src/routes/blocks/block/+page.svelte

```diff
@@ -1,17 +1,17 @@
 <script>
+  import { onMount } from "svelte";
   import { navigating, page } from "$app/stores";
   import { BlockDetails, LatestTransactionsCard } from "$lib/components";
   import { duskAPI } from "$lib/services";
   import { appStore } from "$lib/stores";
   import { createDataStore } from "$lib/dusk/svelte-stores";
-  import { onNetworkChange } from "$lib/lifecyles";
 
   const dataStore = createDataStore(duskAPI.getBlock);
   const payloadStore = createDataStore(duskAPI.getBlockDetails);
 
   const getBlock = () => {
-    dataStore.getData($appStore.network, $page.url.searchParams.get("id"));
-    payloadStore.getData($appStore.network, $page.url.searchParams.get("id"));
+    dataStore.getData($page.url.searchParams.get("id"));
+    payloadStore.getData($page.url.searchParams.get("id"));
   };
 
   const updateData = () => {
```

Comment:
`updateData` can be removed and the `onMount` hook can directly call `getBlock`

Reply:
After looking at this closer the `onMount` hook and `getBlock` can be removed and directly call 
`
dataStore.getData($page.url.searchParams.get("id"));
payloadStore.getData($page.url.searchParams.get("id"));
`